qulice-checkstyle/pom.xml

```diff
@@ -155,6 +155,7 @@ OF THE POSSIBILITY OF SUCH DAMAGE.
                 <exclude>dependencies:org.antlr</exclude>
                 <exclude>xml:/src/main/resources/com/qulice/checkstyle/checks.xml</exclude>
                 <exclude>xml:/src/test/resources/com/qulice/checkstyle/ChecksTest/.*</exclude>
+                <exclude>duplicatefinder:</exclude>
```

Comment:
After updating dependencies, I got the following error (I believe it is false positive, as this is some metadata, not Java class)
```
Warning:  Found duplicate and different classes in [info.picocli:picocli:4.6.1,org.slf4j:slf4j-api:2.0.0-beta1] :
Warning:    META-INF.versions.9.module-info
```
Unfortunately, all my other attempts (such as specifying `duplicatefinder: META-INF.versions.9.module-info`) were not successful at suppressing this warning.